src/main/java/racingGame/RacingGame.java

```diff
@@ -48,4 +47,26 @@ private void saveRound() {
 
         rounds.add(new Round(positions));
     }
+
+    public List<String> winners() {
+        int max = maxPosition();
+        List<String> result = new ArrayList<>();
+
+        for (Car car : cars) {
+            if (car.position() == max) {
```

Comment:
객체의 값을 꺼내지 말고 메시지를 보내 구현해 본다.
객체 지향적인 사고에 정말 중요한 부분인 만큼 앞으로 객체의 값을 꺼내려고 할 때 의식적으로 메시지를 보내기 위한 사고를 하기 위해 노력해 본다.

```suggestion
            if (car.isSame(max)) {
```